code/game/atoms.dm

```diff
@@ -33,6 +33,8 @@
 	var/default_alpha = 255
 	var/default_scale = 1
 
+	var/list/filter_data //For handling persistent filters
```

Comment:
We were just arguing about memory footprint this morning. Adding a new list to all atoms is a terrible idea.

Extensions can manage filters for the few things that use them, no new list on atoms

Reply:
> We were just arguing about memory footprint this morning. Adding a new list to all atoms is a terrible idea.
> 
> Extensions can manage filters for the few things that use them, no new list on atoms

Reverted